api/src/app.module.ts

```diff
@@ -39,14 +40,16 @@ import { LoggingModule } from '@api/modules/logging/logging.module';
     { provide: APP_GUARD, useClass: JwtAuthGuard },
     { provide: APP_FILTER, useClass: AllExceptionsFilter },
     ContactMailer,
+    SQLAdapter, // Weird but works, let's move on. The logger was useful so the utility object became a class
```

Comment:
This goes against the current code organization, it should be a injectable class that belongs to some module, so it follows a certain convention, so other can pick this up easily